core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -172,6 +172,13 @@
         CREATED, OPTIMIZED, OPTIMIZED_AND_VALIDATED
     }
 
+    public enum PlanStats
+    {
+        EXISTING,
```

Comment:
What does EXISTING mean?

Reply:
I am happy to improve the code (or reimplement if needed), provided that there is an agreement as to the end-user visible behavior.